### ISSUE TITLE
Query() vs QueryWithTimeSequenceInformation()

### ISSUE THREAD

**Issue by shuvuu** [AddictedCS/soundfingerprinting#61]
Hello, I am new to your project. And I have some questions. What are their difference? 
I extracted chunks from original mp3 file. (song_00_30 means same ffmpeg's args: -ss 0 -t 30) And run test against each other. But results are not good as expected. Why these results are very different? For example MatchedFingerprints 21 vs 1800+ ? 

```
Fingerprinting duration: 00:00:00 Filename: song_00_30.mp3, Duration: 00:00:30
Fingerprinting duration: 00:00:00 Filename: song_00_40.mp3, Duration: 00:00:40
Fingerprinting duration: 00:00:00 Filename: song_00_50.mp3, Duration: 00:00:50
Fingerprinting duration: 00:00:00 Filename: song_10_30.mp3, Duration: 00:00:30
Fingerprinting duration: 00:00:00 Filename: song_10_40.mp3, Duration: 00:00:40
Fingerprinting duration: 00:00:00 Filename: song_10_50.mp3, Duration: 00:00:49
Fingerprinting duration: 00:00:03 Filename: song_orignal.mp3, Duration: 00:03:26

**Searching:       secondsToAnalyze = 30, startAtSecond = 0** 

**Input file: song_00_30.mp3**
  QueryWithTimeSequenceInformation().Result
   song_orignal.mp3, MatchedFingerprints: 21, SequenceStart: 0.928882438316401, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_00_30.mp3, MatchedFingerprints: 18, SequenceStart: 0.928882438316401, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_10_30.mp3, MatchedFingerprints: 9, SequenceStart: 2.7866473149492, Confidence: 0.495867768595041, SequenceLength: 13.933236574746 
  Query().Result
   song_orignal.mp3, MatchedFingerprints: 1833 
   song_00_30.mp3, MatchedFingerprints: 1786 
   song_00_40.mp3, MatchedFingerprints: 1786 
   song_00_50.mp3, MatchedFingerprints: 1786 
   song_10_30.mp3, MatchedFingerprints: 602 
   song_10_40.mp3, MatchedFingerprints: 602 
   song_10_50.mp3, MatchedFingerprints: 602 

**Input file: song_00_40.mp3**
  QueryWithTimeSequenceInformation().Result
   song_00_30.mp3, MatchedFingerprints: 18, SequenceStart: 0.928882438316401, Confidence: 0.926768721555647, SequenceLength: 26.0087082728592 
   song_orignal.mp3, MatchedFingerprints: 17, SequenceStart: 0.928882438316401, Confidence: 0.959867604468349, SequenceLength: 26.9375907111756 
   song_10_30.mp3, MatchedFingerprints: 12, SequenceStart: 0.928882438316401, Confidence: 0.595779892428631, SequenceLength: 16.7198838896952 
  Query().Result
   song_00_30.mp3, MatchedFingerprints: 1959 
   song_00_40.mp3, MatchedFingerprints: 1959 
   song_00_50.mp3, MatchedFingerprints: 1959 
   song_orignal.mp3, MatchedFingerprints: 1550 
   song_10_30.mp3, MatchedFingerprints: 836 
   song_10_40.mp3, MatchedFingerprints: 836 
   song_10_50.mp3, MatchedFingerprints: 836 

**Input file: song_00_50.mp3**
  QueryWithTimeSequenceInformation().Result
   song_00_30.mp3, MatchedFingerprints: 21, SequenceStart: 0.928882438316401, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_orignal.mp3, MatchedFingerprints: 19, SequenceStart: 0.928882438316401, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_10_30.mp3, MatchedFingerprints: 10, SequenceStart: 0, Confidence: 0.628099173553719, SequenceLength: 17.6487663280116 
  Query().Result
   song_orignal.mp3, MatchedFingerprints: 1966 
   song_00_30.mp3, MatchedFingerprints: 1907 
   song_00_40.mp3, MatchedFingerprints: 1907 
   song_00_50.mp3, MatchedFingerprints: 1907 
   song_10_30.mp3, MatchedFingerprints: 579 
   song_10_40.mp3, MatchedFingerprints: 579 
   song_10_50.mp3, MatchedFingerprints: 579 

**Input file: song_10_30.mp3**
  QueryWithTimeSequenceInformation().Result
   song_00_40.mp3, MatchedFingerprints: 18, SequenceStart: 10.2177068214804, Confidence: 0.991735537190083, SequenceLength: 27.866473149492 
   song_10_30.mp3, MatchedFingerprints: 17, SequenceStart: 4.644412191582, Confidence: 0.793388429752066, SequenceLength: 22.2931785195936 
   song_00_30.mp3, MatchedFingerprints: 12, SequenceStart: 10.2177068214804, Confidence: 0.59504132231405, SequenceLength: 16.7198838896952 
  Query().Result
   song_10_30.mp3, MatchedFingerprints: 1596 
   song_10_40.mp3, MatchedFingerprints: 1596 
   song_10_50.mp3, MatchedFingerprints: 1596 
   song_00_40.mp3, MatchedFingerprints: 1274 
   song_00_50.mp3, MatchedFingerprints: 1274 
   song_orignal.mp3, MatchedFingerprints: 1241 
   song_00_30.mp3, MatchedFingerprints: 825 

**Input file: song_10_40.mp3**
  QueryWithTimeSequenceInformation().Result
   song_00_40.mp3, MatchedFingerprints: 17, SequenceStart: 10.2177068214804, Confidence: 0.991735537190083, SequenceLength: 27.866473149492 
   song_orignal.mp3, MatchedFingerprints: 16, SequenceStart: 10.2177068214804, Confidence: 0.991735537190083, SequenceLength: 27.866473149492 
   song_10_30.mp3, MatchedFingerprints: 15, SequenceStart: 1.8577648766328, Confidence: 0.859504132231405, SequenceLength: 24.1509433962264 
   song_00_30.mp3, MatchedFingerprints: 12, SequenceStart: 10.2177068214804, Confidence: 0.628099173553719, SequenceLength: 17.6487663280116 
  Query().Result
   song_00_40.mp3, MatchedFingerprints: 1752 
   song_00_50.mp3, MatchedFingerprints: 1752 
   song_10_30.mp3, MatchedFingerprints: 1662 
   song_10_40.mp3, MatchedFingerprints: 1662 
   song_10_50.mp3, MatchedFingerprints: 1662 
   song_orignal.mp3, MatchedFingerprints: 1491 
   song_00_30.mp3, MatchedFingerprints: 1166 

**Input file: song_10_50.mp3**
  QueryWithTimeSequenceInformation().Result
   song_10_30.mp3, MatchedFingerprints: 22, SequenceStart: 0.928882438316401, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_00_40.mp3, MatchedFingerprints: 18, SequenceStart: 10.2177068214804, Confidence: 0.991735537190083, SequenceLength: 27.866473149492 
   song_orignal.mp3, MatchedFingerprints: 14, SequenceStart: 11.1465892597968, Confidence: 0.958677685950413, SequenceLength: 26.9375907111756 
   song_00_30.mp3, MatchedFingerprints: 11, SequenceStart: 10.2177068214804, Confidence: 0.628099173553719, SequenceLength: 17.6487663280116 
  Query().Result
   song_10_30.mp3, MatchedFingerprints: 1802 
   song_10_40.mp3, MatchedFingerprints: 1802 
   song_10_50.mp3, MatchedFingerprints: 1802 
   song_orignal.mp3, MatchedFingerprints: 1593 
   song_00_40.mp3, MatchedFingerprints: 1040 
   song_00_50.mp3, MatchedFingerprints: 1040 
   song_00_30.mp3, MatchedFingerprints: 651 

**Input file: song_orignal.mp3**
  QueryWithTimeSequenceInformation().Result
   song_00_40.mp3, MatchedFingerprints: 21, SequenceStart: 0.928882438316401, Confidence: 1.02606537029375, SequenceLength: 28.7953555878084 
   song_00_30.mp3, MatchedFingerprints: 20, SequenceStart: 0.928882438316401, Confidence: 0.959867604468349, SequenceLength: 26.9375907111756 
   song_orignal.mp3, MatchedFingerprints: 18, SequenceStart: 0.928882438316401, Confidence: 0.959867604468349, SequenceLength: 26.9375907111756 
   song_10_30.mp3, MatchedFingerprints: 12, SequenceStart: 0.928882438316401, Confidence: 0.595779892428631, SequenceLength: 16.7198838896952 
  Query().Result
   song_orignal.mp3, MatchedFingerprints: 2282 
   song_00_30.mp3, MatchedFingerprints: 2065 
   song_00_40.mp3, MatchedFingerprints: 2065 
   song_00_50.mp3, MatchedFingerprints: 2065 
   song_10_30.mp3, MatchedFingerprints: 859 
   song_10_40.mp3, MatchedFingerprints: 859 
   song_10_50.mp3, MatchedFingerprints: 859 

```

**Comment by AddictedCS** [comment]
`QueryWithTimeSequenceInformation` uses heuristics to find the exact location of the query in the samples. The number of fingerprints in the result set should not be a concern. What matters is the matched result track not the fingerprint counts.

**Comment by AddictedCS** [comment]
Fixed in 3.0.0. Its now available on NuGet 3.0.0-beta.

**Comment by AddictedCS** [comment]
Closing due to inactivity.
